README.md

```diff
@@ -36,43 +36,31 @@ To this end, how can we easily access our API key without opening ourselves up t
 
 There are many ways to store sensitive information but we will go with this method. 
 
-#### Create a `.secret/` directory
+#### Create a `secret/` directory
 
-This will create a new folder in your project directory where you can store files for any of the API information you have. If you were to push this project to GitHub (ie. share it online) we could put this file in our .gitignore so it would not be shared. To this end, you will be the only one with access to this folder and your API key will be secure!
+Click on the Illumidesk logo in the top left corner while holding the `cmd` key (`ctrl` for Windows) to open the window in a new tab. This is the root of your Illumidesk directory. In this new window, click `New` in the top right corner followed by `Folder`. Find the folder titled `Untitled Folder`, select the checkbox directly to the left of it, scroll back to the top of the page and click `Rename`. Rename the folder to `secret`.
 
+#### Create `.json` file
 
-```python
-!mkdir .secret/
-```
-
-#### Check to make sure it was created
-
-dot files won't show up with just `ls` you must use the show all command as well `ls -a`
-
-
-```python
-!ls -a
-```
+Click on the newly created `secret` folder. Now we want to create a `.json` file to securely store our API key. Once again, click `New` in the top right corner of the page followed by `Text File`.
 
-#### Secure API Key in .json file
+In this file, let's create a dictionary representing the API key that looks something like this:
 
-Next we will create a `.json` file to store our API key. In the cell below, replace `<YOUR_API_KEY>` with the API key you were given by yelp. **Make sure you do not remove any quotations**.
+`{"api_key": "<YOUR_API_KEY>"}`
 
+**NOTE**: Double quotes are important! 
 
-```python
-!echo '{"api_key": "<YOUR_API_KEY>"}' >> yelp_api.json
-```
+Replace `<YOUR_API_KEY>` with the API key you were given by yelp.
 
-#### Move `yelp_api.json` to `.secret/`
+#### Save as `yelp_api.json`
 
-Finally we can move the `.json` file storing our API key into our secure `.secret/` folder. 
+Once you have stored your API key in the text file, go back to your `secret` folder so we can rename the file. Click on the checkbox next to the text file (`untitled.txt`) then select `Rename` at the top. Name the file `yelp_api.json`.
 
 
-```python
-!mv yelp_api.json .secret/
-```
+#### Import API key to jupyter notebook
+Now that we have the API key stored securely (mostly) in our `secret` folder, let's import it into our jupyter notebook.
 
-The function below opens the `.json` file of the given `path`. We have provided the path to your `yelp_api.json` file below.
+If we remember that our file is just a regular JSON file, open the file and pull out the appropriate information from the `../secret/yelp_api.json file` (The path has been provided for you below)
```

Comment:
Just want to confirm here, did you test out that these instructions worked? I don't see a reason why they _wouldn't_, but just want to be sure

Reply:
Yes this method worked for both this Codealong and the Reading Documentation lesson